docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -0,0 +1,484 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:AccessControl: xref:AccessControl[AccessControl]
+:Ownable: xref:Ownable[Ownable]
+:src5: https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-5.md[SRC5]
+:inner-src5: xref:api/introspection.adoc#ISRC5[SRC5 ID]
+:_set_role_admin: xref:#AccessControl-_set_role_admin[_set_role_admin]
+
+= Access Control
+
+[.readme-notice]
+NOTE: This document is better viewed at https://docs.openzeppelin.com/contracts/api/access
+
+This directory provides ways to restrict who can access the functions of a contract or when they can do it.
+
+- {Ownable} is a simple mechanism with a single "owner" role that can be assigned to a single account.
+This mechanism can be useful in simple scenarios, but fine grained access needs are likely to outgrow it.
+- {AccessControl} provides a general role based access control mechanism. Multiple hierarchical roles can be created and
+assigned each to multiple accounts.
+
+== Authorization
+
+[.contract]
+[[Ownable]]
+=== `++Ownable++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/access/ownable/ownable.cairo[{github-icon},role=heading-link]
+
+```javascript
+use openzeppelin::access::ownable::Ownable;
+```
+
+`Ownable` provides a basic access control mechanism where an account
+ (an owner) can be granted exclusive access to specific functions.
+
+This module includes the `assert_only_owner` internal to restrict a function to be used only by the owner.
+
+[.contract-index]
+.External Functions
+--
+[.contract-subindex-inherited]
+.OwnableImpl
+
+* xref:Ownable-owner[`++owner(self)++`]
+* xref:Ownable-transfer_ownership[`++transfer_ownership(self, new_owner)++`]
+* xref:Ownable-renounce_ownership[`++renounce_ownership(self)++`]
+--
+
+[.contract-index]
+.Internal Functions
+--
+[.contract-subindex-inherited]
```

Comment:
To avoid subsections being grayed out, you can remove `[.contract-subindex-inherited]`. I've updated the CSS so that the only difference is the opacity.

Reply:
No strong defense, but to add the comment, I feel it may look better if we let the subindex grayed (not the list element), because being both black now, looks weird to me in the UI.

I don't have a problem with all being grayed out for consistency anyway, but let me know what do you think @martriay  (this will affect other PRs as well)

Reply:
@ericnordelo not sure what you mean, can you screenshot the two alternatives to compare?

Reply:
I just realized that if what you meant is to remove all the grayed-out sections (make everything black), that makes sense to me. I was thinking of just putting the first section black, and the rest grayed out. Is that what you meant?

Reply:
yes all of them

Reply:
that looks right